apps/condo/domains/common/components/PopupSmart.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react'
+import get from 'lodash/get'
+import cookie from 'js-cookie'
+import getConfig from 'next/config'
+import { useOrganization } from '@open-condo/next/organization'
+
+const PopupSmart = (): React.ReactElement | null => {
+    const { link } = useOrganization()
+    const role = get(link, 'role.nameNonLocalized')
+
+    const { publicRuntimeConfig: { popupSmartUrl } } = getConfig()
+
+    // watch for user role update. This is the way to tell popupsmart local user context
+    useEffect(() => {
+        if (role) {
```

Comment:
Maybe there is a race condition. it may be already loaded

Reply:
I don't quite understand what is the race condition about?

Reply:
The script can be loaded before you set the cookies. But you already fixit